submit_a_new_code.md

```diff
@@ -8,10 +8,19 @@ comment: >-
 
 ### Requirements
 
-The main requirements for listing a code are that the code includes an
-[open source license](http://choosealicense.com){:target="_blank"}, is
-available via a public repository and includes annotated phase field
-examples.
+The two main requirements for including a code on the front page are
+
+ 1. that at least [one simulation result]({{ site.baseurl
+    }}/simulations/#simulations) is submitted for the benchmark problems using the
+    code in question (see [instructions]({{ site.baseurl
+    }}/simulations/upload))
+
+ 2. and that the submitted benchmark results are reproducible by
+    another researcher.
```

Comment:
I improved this with "the submitted benchmark results are reproducible by another researcher with the code in question."